broker/pkg/coordinator/loader.go

```diff
@@ -197,6 +201,7 @@ func (c *Loader) loadOneConsumer(ctx context.Context, limit int, tuple *Consumer
 					sent++
 				}
 			}
+			tuple.C.AddAvailableBufferSize(-uint32(len(pushLeaseMessages)))
```

Comment:
You should use the sent variable if you want

Reply:
sent is a count per consumer not per bucket, the calls to storage are per bucket and I'll have to update the limit per bucket